app/models/concerns/mergeable.rb

```diff
@@ -8,16 +8,15 @@ module Mergeable
   class_methods do
     def compute_merge_status(attributes)
       new_record = new(attributes)
-      existing_record = with_discarded.find_by(id: attributes['id'])
+      existing_record = with_discarded.find_by(id: attributes["id"])
 
-      case
```

Comment:
I'm really not happy about standardrb deciding to change all case statements to if / elsif statemnts. How would the team feel about overriding that one rule globally?

FWIW there was a lot of push back on that rule when it [landed in Rubocop](https://github.com/rubocop-hq/rubocop/issues/3019).

Reply:
I really much prefer `case` statements over long if/else. I'm 👍 for overriding that rule.